tests/pw/pages/liveChatPage.ts

```
@@ -0,0 +1,92 @@
+import { helpers } from '@utils/helpers';
+
+// selectors
+const liveChatVendor = selector.vendor.vInbox;
+const liveChatCustomer = selector.customer.cLiveChat;
+const singleStoreCustomer = selector.customer.cSingleStore;
+const singleProductCustomer = selector.customer.cSingleProduct;
+
+export class LiveChatPage extends BasePage {
+    constructor(page: Page) {
+        super(page);
+    }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)